minigalaxy/ui/gtk.py

```diff
@@ -0,0 +1,4 @@
+import gi
+
+gi.require_version('Gtk', '3.0')
+from gi.repository import Gtk, Gdk, Gio, GLib, GdkPixbuf  # noqa: E402,F401
```

Comment:
Ah, this is how you solved this. I've been wondering how to do this for a while. Looks like a good solution

Reply:
Yeah, there is a little problem with this solution, though: I couldn't find a way to "forward" the package imports. In other words, I don't think there's a way to allow something like this:
```python
from minigalaxy.ui.gtk.GdkPixbuf import Pixbuf
...
pixbuf = Pixbuf(...)
```
You always have to do it like this:
```python
from minigalaxy.ui.gtk import GdkPixbuf
...
pixbuf = GdkPixbuf.Pixbuf(...)
```
However, it's rarely needed (I've only made a couple changes across the whole codebase), so I think this solution should be good enough for a long while. If someone figures this out, that'd definitely be an improvement. It could probably be done by mirroring the GTK package structure, but I didn't think it was pretty enough. Anyway, as I said, the current solution should be good enough. I'm glad you like it